qemu/qmp.go

```diff
@@ -755,6 +756,23 @@ func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevType, netdevID, ifname
 	return q.executeCommand(ctx, "netdev_add", args, nil)
 }
 
+// ExecuteNetdevAddByFds adds a Net device to a QEMU instance
+// using the netdev_add command by fds and vhostfds. netdevID is the id of the device to add.
+// Must be valid QMP identifier.
+func (q *QMP) ExecuteNetdevAddByFds(ctx context.Context, netdevType, netdevID string, tapFds, vhostFds []string) error {
```

Comment:
Maybe we should use `...FdNames` instead of `...Fds`. This way, we make it clear that we need to pass the fdname that has been sent to QMP by using `GetFd()`